src/middlewared/middlewared/plugins/webdav.py

```diff
@@ -226,6 +227,15 @@ async def upper(self, data):
 
         return data
 
+    @accepts()
```

Comment:
We are missing `returns` decorator and a doc string as this is a public method 😅

Reply:
Meh